app/src/main/java/in/hangang/hangang/data/source/HomeRepository.kt

```diff
@@ -0,0 +1,25 @@
+package `in`.hangang.hangang.data.source
+
+import `in`.hangang.hangang.data.response.CommonResponse
+import io.reactivex.rxjava3.core.Single
+
+class HomeRepository(
+        private val homeLocalDataSource: HomeDataSource,
+        private val homeRemoteDataSource: HomeDataSource
+) : HomeDataSource{
+    override fun getLectureRanking(major: String): Single<CommonResponse> {
+        return Single.never()
```

Comment:
구현을 해야되면은 TODO를 사용하는게 좋아보입니다.
```suggestion
     TODO("not implemented")
```